data-raw/parity_airyards_oracle.py

```
@@ -0,0 +1,46 @@
+SDV_PY = os.environ.get('SDV_PY_ROOT', '/mnt/sdv_repos/sportsdataverse-py')  # sdv-py checkout (or leave importable)
+sys.path.insert(0, SDV_PY)
+from sportsdataverse.cfb import CFBPlayProcess
+RAW = os.environ.get('CFB_RAW_JSON', '/mnt/sdv_repos/cfbfastR-cfb-raw/cfb/json/raw')  # banked ESPN summaries
+OUT = pathlib.Path(__file__).resolve().parents[1] / 'tests' / 'testthat' / 'fixtures' / 'parity'
+fg = json.load(open(OUT / 'fixture_games.json'))
+ids = [g['game_id'] for season in fg.values() for g in season]
+extra = [401752772, 401760418, 401761645, 401757293, 401864577]  # 2025 TA&M-SC, UNLV-HAW, CCU-GAST, LIB-DEL; 2026 NDSU-JVST
+ids = ids + [g for g in extra if g not in ids]
+IN = ['id','text','start.yardsToEndzone','end.yardsToEndzone','pos_team','def_pos_team','homeTeamId','awayTeamId','homeTeamAbbrev','awayTeamAbbrev','statYardage','completion','pass']
+OUTC = ['air_yardsToEndzone','air_yards','yards_after_catch']
+frames = []; missing = []
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)